### PR TITLE
Tweak error throttling

### PR DESCRIPTION
**Pull request by sa-spag**

### CODE REVIEW

File: sidecar/resources.py [hunk 1]
```diff
@@ -160,12 +160,12 @@ def _watch_resource_iterator(label, labelValue, targetFolder, url, method, paylo
 def _watch_resource_loop(mode, *args):
     while True:
         try:
-            # Always wait to slow down the loop in case of exceptions
-            sleep(os.getenv("ERROR_THROTTLE_SLEEP", 5))
             if mode == "SLEEP":
-                listResources(*args)
                 sleep(os.getenv("SLEEP_TIME", 60))
```
Comment (usmonster) [inline sidecar/resources.py:164]: If `SLEEP_TIME` < `ERROR_THROTTLE_SLEEP`, you may want to wait for sleep for the `max` of the two in this case.

File: .circleci/config.yml [hunk 2]
```diff
@@ -71,7 +71,7 @@ jobs:
   test-k8s-1-14:
     machine: true
     environment:
-      KIND_VERSION: v0.7.0
+      KIND_VERSION: v0.8.1
```
Comment (usmonster) [inline .circleci/config.yml:74]: Not required here.

File: .circleci/config.yml [hunk 3]
```diff
@@ -53,7 +53,7 @@ jobs:
   test-k8s-1-13:
     machine: true
     environment:
-      KIND_VERSION: v0.7.0
+      KIND_VERSION: v0.8.1
```
Comment (usmonster) [inline .circleci/config.yml:56]: Not required here.

File: .circleci/config.yml (hunk 2 above)
Comment (sa-spag) [reply .circleci/config.yml:74]: I updated because end-to-end tests were unstable. Seems better now (unless I'm just lucky).

Comment (usmonster) [reply .circleci/config.yml:74]: Yep, I just meant that they seemed to be fine for these specific tests, so don't need to be changed here.

File: .circleci/config.yml (hunk 3 above)
Comment (sa-spag) [reply .circleci/config.yml:56]: I used to have unexpected failure for Kubernetes 1.13 too (e.g. [here](https://app.circleci.com/pipelines/github/blablacar/k8s-sidecar/6/workflows/f3e42f67-a50e-45eb-94d6-1ebf4311e076/jobs/15)), so I'd rather update for all targets.

**Pull request merged**